laravel-app/database/seeders/UserPostsTableSeeder.php

```diff
@@ -17,14 +17,16 @@ public function run(): void
                 'user_id' => 1,
                 'content' => 'The quick brown fox jumps over the lazy dog :O',
                 'created_at' => now(),
+                'updated_at' => now(),
             ],
             [
                 'user_id' => 1,
                 'content' => 'Time is gold, spend it wisely :)',
                 'created_at' => now(),
+                'updated_at' => now(),
             ],
         ];
 
-        DB::table('user_posts')->insert($user_posts);
+        UserPost::insert($user_posts);
```

Comment:
Is this working? AFAIK if you want to create data using eloquent it should be create() right?

Reply:
Insert works sir